pmlc/rt/vulkan/wrappers.cc

```diff
@@ -83,9 +84,10 @@ void run(void *vkInvocation) {
 #define BIND_BUFFER_IMPL(_name_, _type_)                                       \
   void _mlir_ciface_bindBuffer##_name_(                                        \
       void *vkInvocation, DescriptorSetIndex setIndex, BindingIndex bindIndex, \
-      uint32_t bufferByteSize, ::UnrankedMemRefType<_type_> *unrankedMemRef) { \
+      uint32_t bufferByteSize, BUFFER_COPY_MODE bufferCopyMode,                \
```

Comment:
Please make bufferCopyMode an int32_t (matching the call op), and then cast it before passing it to bindBuffer, so that the ABI is declared the same way on both the caller and callee.

Reply:
Fixed, please check.